src/racing/actors.c

```diff
@@ -2816,88 +1725,88 @@ s32 func_8029F69C(Player *player, struct Actor *actor) {
         player->pos[0] += x_dist * temp_f2 * 0.5f;
         player->pos[2] += z_dist * temp_f2 * 0.5f;
     }
-    return 1;
+    return TRUE;
 }
 
-s32 func_8029FB80(Player *arg0, struct Actor *arg1) {
+bool is_collision_between_item_player(Player *arg0, struct Actor *arg1) {
```

Comment:
`query_collision_player_vs_item_aabb`

_item_ or _actor_ ?

Reply:
it's all actor who are like item so shells and banana

Reply:
and what the aabb mean ?

Reply:
I rename it query_collision_player_vs_actor_item for now